### ISSUE TITLE
Block all Cookies

### ISSUE THREAD

**Issue by JonathanKrug** [duckduckgo/iOS#590]
Hi, at this time the DDG App allows all cookies (except those who are blocked by tracking list). Cookies are a danger for Privacy. It would be great if there would be a possibility in the App-Settings to block all Cookies like in Safari!

**Comment by brindy** [comment]
Hi, thanks for your suggestion and sorry for the delay in replying.

I'm afraid we don't plan to implement this for a couple of key reasons:

1) Functionality of many websites depends on cookies - disabling them en masse will break a lot of the web and even as an option we don't believe that will give a good enough user experience that is worth having.

2) WKWebView doesn't actually allow us to do this.  In order to do it we'd either need to be able to observe and modify the HTTP requests of individual resources or observe cookies reliably.  Unfortunately, neither of those things are are currently possible.  In the latter case, observing cookies only works when you have a single instance of WKWebView, but we support multiple tabs and therefore have multiple instances of WKWebView active at once.

Thanks for reaching out to us, your feedback has been noted. Please feel free to open a new issue if you find any other problems and thanks for your continued use of our app!
